final-project/docker-compose.yml

```diff
@@ -0,0 +1,45 @@
+services:
```

Comment:
it didn't work for me with different errors, but I think it's close to be ready. great effort, thanks! 

backend     | sh: 1: nodemon: not found
backend exited with code 127
frontend    | npm ERR! Missing script: "client"
frontend    | npm ERR! 
frontend    | npm ERR! To see a list of scripts, run:
frontend    | npm ERR!   npm run